### ISSUE TITLE
use github pages to host official helm chart repository

### ISSUE THREAD

**Issue by jackfrancis** [kubernetes-sigs/cloud-provider-azure#1418]
#### What would you like to be added:

This is a proposal to consider using the github pages feature to host the helm chart repo, rather than hosting it in a static directory in the repo. It looks like there is a `gh-pages` branch that hasn't been updated in over a year, so maybe we can use it for this?

I've had success in other projects using this github action to accomplish this goal:

https://github.com/helm/chart-releaser-action

#### Why is this needed:

Not 100% required, but would make helm chart downloads easier for users. Compare:

- `helm install --repo https://kubernetes-sigs.github.io/cloud-provider-azure/`

to

- `helm install --repo https://raw.githubusercontent.com/kubernetes-sigs/cloud-provider-azure/master/helm/repo`

The former (which is what a gh-pages-backed repo would look like) is cleaner IMO.

**Comment by jaanhio** [comment]
hello @jackfrancis , i am happy to work on this if it isn't taken yet. 

but one clarification i have is, is the intention to remove the existing hugo pages and replace it with hosting of helm chart repo or have **both** the hugo pages and helm chart repo hosted together?

**Comment by jackfrancis** [comment]
I don't believe you can get both (the chart-releaser-action takes over the entire hugo implementation as far as I can tell).

cc @lzhecheng @feiskyer for permissions to overwrite the existing github pages content

**Comment by bridgetkromhout** [comment]
@jackfrancis This repo's ghpages branch is in use with some current updates now: https://github.com/kubernetes-sigs/cloud-provider-azure/tree/gh-pages

**Comment by Sajiyah-Salat** [comment]
is this pr still valid?

**Comment by feiskyer** [comment]
yes, better to host with github pages, but we need to check how to host both charts and website together.

/remove-lifecycle stale
